justin-tan/week7/wed/flickr-search/js/main.js

```diff
@@ -0,0 +1,263 @@
+console.log(`flickr search`);
+
+const FLICKR_BASE_URL = 'https://www.flickr.com/services/rest/';
+const FLICKR_API_KEY = '2f5ac274ecfac5a455f38745704ad084';
+
+let pageNum = 1
+let query = ''
+
+$( function(){
+
+    $('#query').focus(); // so we can test easily by pressing enter
+
+    // EITHER a click of the button OR (yessss) pressing enter in an input
+    // will trigger the form submit
+    $('#searchForm').on('submit', function( ev ){
+        ev.preventDefault(); // stop the form from reloading the page
+
+        query = $('#query').val();
+
+        getSearchResults( query )
+
```

Comment:
you should add a `show()` and `hide()` function here for the previous page and next page so they only show when a query has been entered

Reply:
True that. I'll give that a go